script/restore.sh

```diff
@@ -1,29 +1,47 @@
-#!/bin/bash -e
-BDIR=$(readlink -f "$1")
```

Comment:
why not a -y so it can be run non-interactively?

foreman-admin and others would just run 'restore -y'